tests/failpoints/cases/test_snap.rs

```diff
@@ -478,7 +478,6 @@ fn test_gen_snapshot_with_no_committed_entries_ready() {
 #[test]
 fn test_cancel_snapshot_generating() {
     let mut cluster = new_node_cluster(0, 5);
-    cluster.cfg.server.snap_max_write_bytes_per_sec = ReadableSize(10);
```

Comment:
It can fail the case because generating snapshot becomes too slow. It can pass before the PR because the configuration doesn't make any effects exactly.